From rpurdie@rpsys.net Mon Nov 14 10:09:23 2005
From: Richard Purdie <rpurdie@rpsys.net>
To: Greg KH <gregkh@suse.de>
Date: Mon, 14 Nov 2005 17:46:56 +0000
Subject: USB: OHCI lh7a404 platform device conversion fixup
Message-Id: <1131990416.7590.6.camel@localhost.localdomain>

Fix an error in the OHCI lh7a404 driver after the platform device
conversion.

Signed-off-by: Richard Purdie <rpurdie@rpsys.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-lh7a404.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/host/ohci-lh7a404.c
+++ gregkh-2.6/drivers/usb/host/ohci-lh7a404.c
@@ -219,7 +219,7 @@ static int ohci_hcd_lh7a404_drv_probe(st
 
 static int ohci_hcd_lh7a404_drv_remove(struct platform_device *pdev)
 {
-	struct usb_hcd *hcd = platform_get_drvdata(dev);
+	struct usb_hcd *hcd = platform_get_drvdata(pdev);
 
 	usb_hcd_lh7a404_remove(hcd, pdev);
 	return 0;
